times_reader/main.py

```diff
@@ -59,7 +59,6 @@ def read_mappings(filename: str) -> List[datatypes.TimesXlMap]:
                 # Uppercase and validate tags:
                 if xl_name.startswith("~"):
                     xl_name = xl_name.upper()
-                    assert datatypes.Tag.has_tag(xl_name), f"Tag {xl_name} not found"
```

Comment:
I'm curious why this assertion is no longer needed?

Reply:
It was either remove this or add the 'artificial' tables to datatypes.Tag. Not sure I made the right choice with hindsight

Reply:
Opened issue #37